### PR TITLE
add mlcourse.ai ML

### PR DESCRIPTION
**Pull request by Yorko**
Add a large, open and free machine learning course – articles, Jupyter notebooks, assignments, projects, tutorials, Kaggle Inclass competitions and active community OpenDataScience.

### CODE REVIEW

**Comment by Yorko** [comment]
Now it's OK :)

File: README.md [hunk 1]
```diff
@@ -183,6 +183,7 @@ The number after the course name stands for the year in which the course was mad
 - [Artificial intelligence for robotics](https://www.udacity.com/course/artificial-intelligence-for-robotics--cs373)
 - [Neural networks for machine learning](https://www.coursera.org/learn/neural-networks)
 - [Machine learning in Python with scikit-learn](https://github.com/justmarkham/scikit-learn-videos#readme)
+- [mlcourse.ai](http://mlcourse.ai) - open Machine Learning course by OpenDataScience
```
Comment (nikivdev) [inline README.md:186]: Uppercase open.

Also, no need to create new PR, just add a commit and we merge this.

Comment (nikivdev) [reply README.md:186]: Also it needs to end in full stop.

**Comment by Yorko** [comment]
OK, I don't care anymore.
If you want to add this cool course in your list, you can do so.

**Comment by nikivdev** [comment]
> you can do so

Done. 


> I don't care anymore

The list starts with a link to contributing guidelines which you obviously haven't read. It takes maybe 5 seconds to read it.

**Comment by Yorko** [comment]
Thank you!

**Pull request closed without merging**